EstateAdministrationUI/Common/Helpers.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EstateAdministrationUI.Common
+{
+    using System.Linq.Expressions;
+    using Areas.Estate.Models;
+    using Microsoft.AspNetCore.Http;
+    using Shared.Logger;
+    using BusinessLogic.Common;
+
+    public class Helpers
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=6070534)